releases/7.6.0.md

```diff
@@ -302,7 +303,10 @@ Minor incompatibilities:
     3. If it exists, remove `${project_name}/webpack/webpack-user-config.js`
 
 10. In settings.py:
-    1. Remove `"compressor"` from `INSTALLED_APPS`
+    1. In `INSTALLED_APPS`:
+        - remove `"compressor"`
+        - add `"arches.app"`
+        - Reorder anything also present in `ARCHES_APPLICATIONS` to a position before `"arches.app"` to preserve their higher preference with respect to templates and static files.
```

Comment:
We're spawning projects' `settings.py` with `arches.app` somewhere in the middle. And I can forsee a huge headache caused by needing to ensure `arches.app` goes at the end of installed apps. I can't imagine a scenario where a user would want arches-core templates to overwrite all arches_applications templates (+ static files).

I like this pattern better than `inject()`-ing inside `apps.py`. IMO it would be worth adding a small bit of logic in `settings.py` to append this to `INSTALLED_APPS`

Reply:
Nice idea!